lib/actions-oclif/env/rm.ts

```diff
@@ -0,0 +1,109 @@
+/**
+ * @license
+ * Copyright 2019 Balena Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Command, flags } from '@oclif/command';
+import { stripIndent } from 'common-tags';
+
+import { CommandHelp } from '../../utils/oclif-utils';
+
+interface FlagsDef {
+	device: boolean;
+	yes: boolean;
+}
+
+interface ArgsDef {
+	id: number;
+}
+
+export default class EnvRmCmd extends Command {
+	public static description = stripIndent`
+		Use this command to remove an environment variable from an application
+		or device.
```

Comment:
The `cli.markdown` file is printing only "or device", skipping the first line (see the other review comment). If you check the similar `add.ts` file that implements `env add` with oclif, you'll see a duplicate first line:
```
	public static description = stripIndent`
		Add an environment or config variable to an application or device.

		Add an environment or config variable to an application or device, as selected
		by the respective command-line options.
```
It's on purpose, because the oclif help output will print the first line at the very top, and the following lines under "Description".
```
$ balena help env add
Add an environment or config variable to an application or device.

USAGE
...
DESCRIPTION
  Add an environment or config variable to an application or device, as selected
  by the respective command-line options.
```

In addition to the markdown, please check that the following produces the expected output:
```
# 'help -v' should print lines for env add, env rename, env rm, envs
balena help -v
balena help env rm
```